test/cmd/newapp.sh

```diff
@@ -27,9 +27,9 @@ os::cmd::try_until_failure 'oc get istag php:latest -n openshift'
 os::cmd::expect_failure 'oc new-app --image-stream=openshift/php https://github.com/sclorg/cakephp-ex'
 
 # should succeed and create the php:latest tag in the current namespace
-os::cmd::expect_success 'oc new-app --docker-image=library/php https://github.com/sclorg/cakephp-ex --strategy=source'
-os::cmd::try_until_success 'oc get istag php:latest -n test-imagestreams'
-os::cmd::expect_success 'oc create istag php:latest --from=openshift/php:7.1 -n openshift'
+os::cmd::expect_success 'oc new-app --docker-image=quay.io/centos7/php-72-centos7 https://github.com/sclorg/cakephp-ex --strategy=source --name=php'
```

Comment:
If it helps, here's where the an official UBI imagestream comes from:

registry.access.redhat.com/ubi7/php-73

https://catalog.redhat.com/software/containers/ubi7/php-73/5da0228fdd19c70159f4601b

7.2 is already EOL, 7.3 is next and 7.4 is only published for ubi8 currently.

Reply:
I see in 9dc7e4bb750ea7f2e9339a39649e428cf6a68eda you are using ubi8. In that case, registry.access.redhat.com/ubi8/php-74 will give you the most legs. :)

Reply:
Updated quay.io images to registry.access.redhat.com. Its a mix of UBI7/UBI8 now, we can improve this later on